software/controller/lib/debug/var_cmd.cpp

```diff
@@ -48,17 +50,17 @@
 // system starting with 0.  The Python code can read them
 // all out until it gets an error code indicating that the
 // passed ID is invalid.
-DbgErrCode VarCmd::GetVarInfo(CmdContext *context) {
+ErrorCode VarHandler::GetVarInfo(Context *context) {
 
   // We expect a 16-bit ID to be passed
-  if (context->req_len < 3)
-    return DbgErrCode::MISSING_DATA;
+  if (context->request_length < 3)
+    return ErrorCode::kMissingData;
 
-  uint16_t vid = u8_to_u16(&context->req[1]);
+  uint16_t var_id = u8_to_u16(&context->request[1]);
 
-  const auto *var = DebugVar::FindVar(vid);
+  const auto *var = DebugVar::FindVar(var_id);
   if (!var)
-    return DbgErrCode::BAD_VARID;
+    return ErrorCode::kUnknownVariable;
 
```

Comment:
I wonder if we could define this format using protobuffers like we have for the main data protocol? I would not be surprised if it had an implementation for python as well, so we might avoid hard-coding this type of stuff on both ends?

Reply:
We can make a ticket for it and come back to it later?

Reply:
#1073